enum.go

```diff
@@ -271,9 +271,11 @@ const (
 )
 
 // All ConnectiveEnum as []string
-var AllConnectiveEnum = []string{
-	string(ConnectiveEnumAnd),
-	string(ConnectiveEnumOr),
+func AllConnectiveEnum() []string {
```

Comment:
It looks like the generator didn't get updated to correctly create static global vars for the `all` lists (terraform uses these)

We need to undo the transformation to a function and fix the generator.

Reply:
Rebased to `main` which has https://github.com/OpsLevel/opslevel-go/pull/227